### PR TITLE
Ingest vhs1256b try2

### PR DESCRIPTION
**Pull request by kelle**
Updated JSON files for VHS1256b and scripts for ingesting spectra.

### CODE REVIEW

File: scripts/ingests/ingest_utils.py [hunk 1]
```diff
@@ -1307,3 +1340,36 @@ def ingest_gaia_pms(db, sources, gaia_data, ref):
...
+    except KeyError:
+        mode = None
+    obs_date = header['DATE-OBS']
+    doi = header['REFERENC']
+    data_header = fits.getheader(spectrum_fits_file, 1)
+    w_unit = data_header['TUNIT1']
+    flux_unit = data_header['TUNIT2']
+
+    reference_match = db.query(db.Publications.c.publication).filter(db.Publications.c.doi == doi).table()
+    reference = reference_match['publication'][0]
```
Comment (SherelynA) [inline scripts/ingests/ingest_utils.py:1372]: would this line give an error if there is no doi?  I don't know too much about fits files, and if they come with doi's but if they don't then make we could do a try and except for the reference match  which could also maybe make sense for adding unpublished data ?

Comment (kelle) [reply scripts/ingests/ingest_utils.py:1372]: You are absolutely correct! This script needs a ton of error catching. I will add a little bit and add some TODOs since I really want to get the data ingested ASAP.

**Comment by kelle** [comment]
Need to add original spectra before merging.

File: scripts/ingests/ingest_utils.py (hunk 1 above)
Comment (kelle) [reply scripts/ingests/ingest_utils.py:1372]: I've recorded this todo is issue #297.

File: scripts/ingests/ingest_vhs1256b.py [hunk 2]
```diff
@@ -2,172 +2,229 @@
...
+        .where(db.Spectra.c.reference == 'Gauz15')\
+        .values(original_spectrum=gauz15_spectra)
+    db.engine.execute(stmt)
+
+
+def ingest_nirspec_spectrum(db):
+    nirspec_instrument = [{'name': 'NIRSPEC',
+                           'reference': None}]
+    db.Instruments.insert().execute(nirspec_instrument)
+    # keck_nir_spectrum_file = \
```
Comment (SherelynA) [inline scripts/ingests/ingest_vhs1256b.py:82]: maybe you could get rid of this comment since you already mention the file url?

Comment (SherelynA) [reply scripts/ingests/ingest_vhs1256b.py:82]: besides that everything looks great [happy dancing penguin]

File: data/2mass_j12560183-1257276.json [hunk 3]
```diff
@@ -16,11 +16,33 @@
...
+    "Parallaxes": [
+        {
+            "parallax": 47.2733194934019,
+            "parallax_error": 0.47309374809265137,
+            "adopted": true,
+            "comments": "adopted from primary VHS J1256-1257",
+            "reference": "GaiaDR3"
+        },
+        {
+            "parallax": 78.8,
```
Comment (Will-Cooper) [inline data/2mass_j12560183-1257276.json:41]: Interesting parallax difference between this and Gaia!

File: data/2mass_j12560183-1257276.json [hunk 4]
```diff
@@ -69,22 +179,72 @@
...
+            "wavelength_order": null,
+            "comments": null,
+            "reference": "Gauz15",
+            "other_references": null
+        }
+    ],
     "SpectralTypes": [
         {
-            "spectral_type_string": "L7:: VL-G",
+            "spectral_type_string": "L7 VL-G",
```
Comment (Will-Cooper) [inline data/2mass_j12560183-1257276.json:234]: Why this change?

**Review by Will-Cooper — APPROVED** [review]
Lgtm

File: data/2mass_j12560183-1257276.json (hunk 4 above)
Comment (kelle) [reply data/2mass_j12560183-1257276.json:234]: No one uses the double colons to indicate uncertainty except me.
